vulcan/lib/client/jsx/reducers/vulcan_reducer.ts

```diff
@@ -82,7 +82,7 @@ export default function VulcanReducer(
       };
     case 'SET_WORKFLOW':
       const workflowProjects = action.workflow.projects;
-      if (!workflowProjects || !workflowProjects.includes(action.projectName))
+      if ( workflowProjects===undefined || (workflowProjects !== null && !workflowProjects.includes(action.projectName)) )
```

Comment:
@corps updated this since you reviewed last

Reply:
:+1: